src/Tools/IdeCoreBenchmarks/CloudCache/IdeCoreBenchmarksCloudCacheServiceProvider.cs

```diff
@@ -23,19 +23,6 @@ public IdeCoreBenchmarksCloudCacheServiceProvider()
         }
 
         public AbstractPersistentStorageService Create(IPersistentStorageConfiguration configuration)
-        {
-            return new MockCloudCachePersistentStorageService(
-                configuration, @"C:\github\roslyn", cs =>
-                {
-                    if (cs is IAsyncDisposable asyncDisposable)
-                    {
-                        asyncDisposable.DisposeAsync().AsTask().Wait();
-                    }
-                    else if (cs is IDisposable disposable)
-                    {
-                        disposable.Dispose();
-                    }
-                });
-        }
```

Comment:
andrew informed us that our ICloudCache proxy will only expose IDisposable.  So our consumption code only needs to handle that and not IAsyncDisposable.  This greatly simplifies a ton of code.